src/content/chainlink-automation/overview/automation-economics.mdx

```diff
@@ -7,24 +7,41 @@ whatsnext: { "Automation Architecture": "/chainlink-automation/concepts/automati
 "Supported Networks for Automation": "/chainlink-automation/overview/supported-networks" }
 ---
 
+import { TabsContent } from "@components/Tabs"
+
 ## Cost of using Chainlink Automation
 
 Chainlink Automation only requires an execution fee for transactions onchain. This fee includes the transaction cost, a node operator percentage fee (refer to the formula below), and a small fixed gas overhead accounting for gas between the network and the registry. The percentage fee compensates the Automation Network for monitoring and performing your upkeep. The Automation percentage fee varies by chain and is listed on our [Supported Networks](/chainlink-automation/overview/supported-networks) page.
 
-**Formula for Registry v2.1**
+Fees for upkeeps that are configured for LINK payments have a conversion from native tokens to LINK, and this conversion is not performed for upkeeps configured for native token payments. Otherwise, the fee formula is the same:
+
+{/* prettier-ignore */}
+<TabsContent sharedStore="paymentMethod" client:visible>
+<Fragment slot="tab.1">LINK</Fragment>
+<Fragment slot="tab.2">Native tokens</Fragment>
+<Fragment slot="panel.1">
+**Formula for Registry v2.3**
 
 Fee<sub>LINK</sub> = [tx.gasPrice<sub>Native WEI</sub> * (gasUsed + gasOverhead) * (1 + premium%)]/[LINK/Native<sub>Rate in WEI</sub>]
 
```

Comment:
Need to add the blurb re

On Automation v2.2 and earlier, there is a flat per-transaction fee of 0.01 LINK in the following cases:
- For mainnet transactions on Optimism and Base, to account for L1 transaction costs, ensuring that node operators are fairly compensated
- For testnet transactions on all supported networks, in order to account for node infrastructure costs